maintenance/scripts/setup.sh

```diff
@@ -15,15 +15,11 @@ php artisan config:clear
 chown -R www-data ./storage ./vendor
 chmod -R a+r,a+w ./storage ./vendor ./bootstrap/cache
 
-# copy out new .env files
-cd /var/www/html/frontend
-cp ./admin/.env.example ./admin/.env --preserve=all
-
 cd /var/www/html
-cp ./.env.example ./.env --preserve=all
+cp .env.example .env --preserve=all
```

Comment:
@petertgiles This still seems to create a directory (`.env/.env.example`) instead of copying the file :/.

Looks like it is also happening for the `api/.env.example` as well.

Reply:
okay, so I just moved the`.env` back to `apps/web` so we have one file now. Seems to be working so far... 😅 

[move .env back to apps/web](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5528/commits/5695b38c0bdac0b92c37c4a3e7cb004bf7bd4e7c)